packages/now-cli/src/commands/inspect.js

```diff
@@ -130,35 +129,12 @@ export default async function main(ctx) {
     throw err;
   }
 
-  const {
-    id: finalId,
-    name,
-    state,
-    type,
-    slot,
-    sessionAffinity,
-    url,
-    created,
-    limits,
-    version,
-    routes,
-    readyState,
-  } = deployment;
-
-  const isBuilds = version === 2;
-  const buildsUrl = `/v1/now/deployments/${finalId}/builds`;
-
-  const [scale, events, { builds }] = await Promise.all([
-    caught(
-      now.fetch(`/v3/now/deployments/${encodeURIComponent(finalId)}/instances`)
-    ),
-    type === STATIC
```

Comment:
@TooTallNate Are we still allowing STATIC deployments? Do we need to keep this?

Reply:
There are no longer any "STATIC" type deployments in the system. They were all upgraded to v2 with `@vercel/static` outputs.